samples/CommunityToolkit.Maui.Sample/Pages/Converters/IsNotNullConverterPage.xaml

```diff
@@ -0,0 +1,57 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<pages:BasePage
+    x:Class="CommunityToolkit.Maui.Sample.Pages.Converters.IsNotNullConverterPage"
+    xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+    xmlns:mct="http://schemas.microsoft.com/dotnet/2022/maui/toolkit"
+    xmlns:pages="clr-namespace:CommunityToolkit.Maui.Sample.Pages"
+    xmlns:vm="clr-namespace:CommunityToolkit.Maui.Sample.ViewModels.Converters"
+    x:DataType="vm:IsNotNullConverterViewModel"
+    x:TypeArguments="vm:IsNotNullConverterViewModel">
+
+    <pages:BasePage.Resources>
+        <ResourceDictionary>
+            <mct:IsNotNullConverter x:Key="IsNotNullConverter" />
+        </ResourceDictionary>
+    </pages:BasePage.Resources>
+    <pages:BasePage.Content>
+        <Grid
+            Padding="10,10"
+            HorizontalOptions="Fill"
+            RowDefinitions="Auto, Auto, Auto, Auto, Auto"
```

Comment:
Please do not use Auto if possible

Reply:
Here's some more info about why we should avoid using `Auto` when sizing Grid Rows + Columns:

> When using a Grid, try to ensure that as few rows and columns as possible are set to Auto size. Each auto-sized row or column will cause the layout engine to perform additional layout calculations. Instead, use fixed size rows and columns if possible. Alternatively, set rows and columns to occupy a proportional amount of space with the GridUnitType.Star enumeration value, provided that the parent tree follows these layout guidelines.

https://docs.microsoft.com/xamarin/xamarin-forms/deploy-test/performance?WT.mc_id=mobile-0000-bramin#optimize-layout-performance

Reply:
Thanks, @brminnick, not sure if it's changed in the last few releases of XF but if I didn't use auto and the user changed the device font size everything would be cut off. So I've used auto ever since.